config/routes.rb

```diff
@@ -1,3 +1,13 @@
 Spree::Core::Engine.routes.draw do
   # Add your extension routes here
+  Spree::Core::Engine.add_routes do
```

Comment:
is this `add_routes` needed? I think it works with `routes.draw` and probably `add_routes`  is also deprecated in the latest versions of solidus.